Sources/MongoSwift/APM.swift

```diff
@@ -117,6 +117,64 @@ private protocol CommandEventProtocol {
     var serviceID: BSONObjectID? { get }
 }
 
+#if compiler(>=5.5) && canImport(_Concurrency)
```

Comment:
I think this needs to be 5.5.2 rather than 5.5. the reason is that only the 5.5.2+ compiler allows using concurrency on macOS 10.15. (the 5.5.0 and 5.5.1 compilers require macOS 12+.)